README.md

```diff
@@ -1 +1,78 @@
-# JavaScript-Capstone
\ No newline at end of file
+# JavaScript-Capstone
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# JavaScript Capstone Project
+
+> Recipe Application
+> This application show a list of recipe with details and option to make like for any recipe you want.
+
+## Built With
+
+- HTML&CSS
+- VScode 
+- JS
+
+## Technologies
+
+- webpack.
+- JS6 Module.
+
+## Live Demo 
+
+Go to live demo : 
+https://maenkhraisha.github.io/JavaScript-Capstone/
```

Comment:
- [ ] the live demo is not working as expected as it is showing your readme file, please make sure the live demo is working before submitting your code for review.